### ISSUE TITLE
untitled issue

### ISSUE THREAD

**Issue by chandukona** [sigmaSd/IRust#131]
```In: let a = 5;
IRust: failed to format the error output.
This is a bug in IRust.
Feel free to open a bug-report at https://github.com/sigmaSd/IRust/issues/new with the next text:

original_output:
error: could not find `Cargo.toml` in `/private/var/folders/fr/h0xg5y_j1zggbs3hs0rb7y5c0000gn/T/irust_repls/irust_host_repl_616768c53da74cc9881335d2a4fd0a80` or any parent directory

In:
```

**Comment by sigmaSd** [comment]
Hello, how did you install irust, this path is weird `/private/var/folders/fr/h0xg5y_j1zggbs3hs0rb7y5c0000gn/T/irust_repls`, it should be the temporary folder of you system , so it seem like have an uncommon system setup

**Comment by OnlyCS** [comment]
Same issue when installed through rustup

In: let _ = String::new();
IRust: failed to format the error output.
This is a bug in IRust.
Feel free to open a bug-report at https://github.com/sigmaSd/IRust/issues/new with the next text:

original_output:
    Checking irust_host_repl_095086b20c5b4c549ceaa24ac01aaa8e v0.1.0 (/tmp/irust_repls/irust_host_repl_095086b20c5b4c549ceaa24ac01aaa8e)
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.06s

**Comment by sigmaSd** [comment]
@OnlyCS your issue is different then the op, I pushed a fix here https://github.com/sigmaSd/IRust/commit/2cf79480c4576f90f935578cb6ab13882ce6b987

whats your rust version btw ? (the output now differs it shows  `dev profile` instead of `dev`

**Comment by OnlyCS** [comment]
I'm running rust 1.78.0 nightly. I'll check out that commit soon

**Comment by OxSon** [comment]
I have this issue as well. I'm on MacOS sonoma 14.6.1. See output below (with personal information removed and indicated with `...snip...`:

```
In: :add --path .
 Adding dep [-]
      Adding lis (local) to dependencies
     Locking 1 package to latest compatible version
      Adding lis v0.1.0 (/Users/...snip.../lis)


In: use lis::*;
IRust: failed to format the error output.
This is a bug in IRust.
Feel free to open a bug-report at https://github.com/sigmaSd/IRust/issues/new with the next text:

original_output:
error: failed to get `lis` as a dependency of package `irust_host_repl_15096e4a63114a1d94ef24c8fdbf2d1e v0.1.0 (/private/var/folders/zp/crj3b1gx1zxgjtlbw_dg3wlm0000gn/T/irust_repls/irust_host_repl_15096e4a63114a1d94ef24c8fdbf2d1e)`

Caused by:
  failed to load source for dependency `lis`

Caused by:
  Unable to update /private/Users/...snip.../lis

Caused by:
  failed to read `/private/Users/...snip.../lis/Cargo.toml`

Caused by:
  No such file or directory (os error 2)```

Based on what you said above, I think it's possible the correct path should `/private/tmp/...`?

**Comment by sigmaSd** [comment]
The original issue is a problem in the macos build, fixed in 1.71.29
